self-reflection-1-myerskevin14.qmd

```diff
@@ -0,0 +1,74 @@
+
+8.  After reviewing the syllabus, give some thought to your objectives for this course. Which parts of this course are going to be most useful to your research? What skills do you really want to focus on during the semester? Once you've given this some thought, craft 3-5 SMART objectives for you in this course. SMART objectives are **S**pecific (e.g., I want to make maps in R), **M**easurable (e.g., I want to build *3 maps displaying different elements of a location* in R), **A**chievable (e.g., I want to build 3 maps displaying different elements of a location *using ggplot*), **R**elevant (e.g., I want to build 3 maps displaying *social and ecological* elements of a location using ggplot), and **T**ime-bound (e.g., I want to build 3 maps displaying *social and ecological* elements of a location using ggplot *for the final project*). Try to think about your overall objectives for the course so that we can use to assess your final project. Remember, these objectives should be based in where you are now and where you'd like to be when the course ends. **If you're not sure how to answer this, schedule a meeting with me to chat about it**.
+
+    Map making and becoming familiar with different R tools like quarto will be very helpful in my research. I want to be able to make maps in R. Including features like movement of individuals on the map is another goal I would like to set for myself. I want to build maps that incorporate juvenile movement with relation to their natal territories to show when the young leave their home. For the final project I would like to pick something that has transferable skills to my research like mapping a territory and including land-cover change over time as well as other geological features. I would also like to feel more confident making maps in R with ggplot as well as with ArcGIS. 
+
+9.  Given the grading 'contracts' laid out in the syllabus and your objectives above, what grade would you like to achieve in this course? How will you support yourself in achieving this grade? How can I help?
+
+    I would like to achieve an A in this course, given the provided grading contracts. I hope to support myself by participating in class discussions as well as working on homework assignments to my best ability. I think that the class is laid out in an approachable format and that I will need to ask for help when I need it and not panic until the end of class and leave confused.
+
+10. Are there topics on the syllabus that are particularly important for you? What is missing?
+
+    Mapping in R/GIS is an important part of the class that seems most applicable to my research but am also very interested in learning about Rasters and different statistical modelling techniques. Interpreting movement data is also something I am interested in.
```

Comment:
We have two sessions on movement data planned later in the course, so hopefully those will help! I'm not an expert in movement data though, so when we get there feel free to let us know anything you've used/learned already.